contracts/contracts/TACoApplication.sol

```diff
@@ -282,10 +282,13 @@ contract TACoApplication is
     }
 
     /**
-     * @dev Checks caller is T staking contract
+     * @dev Checks caller is T staking contract or contract Owner
      */
     modifier onlyStakingContract() {
```

Comment:
What about:
```suggestion
    modifier onlyStakingContractOrOwner() {
```

Reply:
I think it's to close to another modifier `onlyOwnerOrStakingProvider`